pyvista/core/objects.py

```diff
@@ -47,10 +47,18 @@ def __init__(self, *args, **kwargs):
             else:
                 raise TypeError(f'Table unable to be made from ({type(args[0])})')
 
+    @staticmethod
+    def _prepare_arrays(arrays):
+        arrays = np.array(arrays, copy=False)
+        if arrays.ndim == 1:
+            return np.reshape(arrays, (1, -1))
+        elif arrays.ndim == 2:
+            return arrays.T
+        else:
+            raise ValueError('Only 1D or 2D arrays are supported by Tables.')
+
     def _from_arrays(self, arrays):
-        if not arrays.ndim == 2:
-            raise ValueError('Only 2D arrays are supported by Tables.')
-        np_table = arrays.T
+        np_table = self._prepare_arrays(arrays)
         for i, array in enumerate(np_table):
             self.row_arrays[f'Array {i}'] = array
         return
```

Comment:
Unnecessary return here and in other spots.  Just noticed it and it's not part of this PR, just figured to clean it up.
```suggestion
```